.gitignore

```diff
@@ -122,12 +122,18 @@ celerybeat.pid
 # Environments
 .env
 .venv
+.vvvenv
```

Comment:
Why are these environments required? Because only 1 environment is needed for now.